File: src/main/java/dirtymaster/fgpt/service/MessagesService.java
package dirtymaster.fgpt.service;

import dirtymaster.fgpt.model.Message;
import lombok.Getter;

import java.util.ArrayList;
import java.util.List;

@Getter
public class MessagesService {
    private final List<Message> messages = new ArrayList<>();

    public void addMessage(Message message) {
        messages.add(message);
    }

    public void clearMessages() {
        messages.clear();
    }
}

File: src/main/java/dirtymaster/fgpt/service/MessagesService.kt
package dirtymaster.fgpt.service

import dirtymaster.fgpt.model.Message

class MessagesService {
    val messages: MutableList<Message> = ArrayList()

    fun addMessage(message: Message) {
        messages.add(message)
    }

    fun clearMessages() {
        messages.clear()
    }
}
